index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="UTF-8" />
   <link href="https://fonts.googleapis.com/css2?family=Material+Icons" rel="stylesheet">
   <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
-  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="/style.css">
```

Comment:
this shouldn't matter should it? (I thought html translated style.css -> /style.css on the root domain)

Reply:
resolved in dm